src/config.ts

```diff
@@ -110,6 +128,9 @@ export function createProxyConfig(option: IProxyOption): IProxyConfig {
     const trustProxy =
         option.trustProxy || loadTrustProxy(process.env.TRUST_PROXY);
 
+    const parsedTags = option.tags || process.env.UNLEASH_TAGS;
+    const tagFilters = mapTagsToFilters(parsedTags);
```

Comment:
I would change this line to simply:
`const tagFilters = option.tags || mapTagsToFilters(process.env.UNLEASH_TAGS);`